includes/definitions/aviat-wtm.yaml

```diff
@@ -9,4 +9,11 @@ over:
 discovery:
     -
         sysObjectID:
+            - .1.3.6.1.4.1.2509.11.1.1.7
```

Comment:
You could just set 
.1.3.6.1.4.1.2509.11.1.1
If it should match all

Reply:
There are also other devices that are not WTM4k family. This is the reason we added only the ones that should be WTM4k